index.css

```
@@ -85,28 +82,49 @@ input[type="text"]::-ms-clear {
 
 .app-menu {
   margin: 0;
-  padding: 0;
+  padding: 0.5rem 0;
 }
 
-.app-menu li {
-  list-style-type: none;
-  margin: 1rem 2rem;
-  cursor: pointer;
-}
-
-.app-menu li img,
-.app-menu li svg {
+.app-menu img,
+.app-menu svg {
   max-height: 1em;
-  margin-right: 2rem;
+  margin: 0 2rem;
   vertical-align: middle;
   width: 1em;
 }
 
-.app-menu li a {
+.app-menu a,
+.app-menu button {
+  background: none;
+  border: none;
   color: inherit;
+  cursor: pointer;
+  display: flex;
   font-size: 20px;
+  padding: 0.5rem 0;
   text-decoration: none;
-  outline: none;
+  width: 100%;
+}
+
+.app-menu a:hover,
+.app-menu button:hover {
+  color: #4c97f5;
+}
+
+.app-menu a:focus,
+.app-menu button:focus {
+  background-color: #ddd;
+  outline: 2px solid #4c97f5;
+  outline-offset: -2px;
+}
+
+.skip-nav-button {
+  color: initial;
+  position: fixed;
+  top: -30px;
+}
+.skip-nav-button:focus {
+  top: 0;
 }
```

What about this, to make the button a little more cohesive with the nav bar? Currently the skip button is covered with the nav button, so the z-index feels the most important out of the below changes

```suggestion
.skip-nav-button:focus {
  padding: 7px 24px;
  top: 7px;
  z-index: 100
}
```

---

Sure, thanks!